File: javascript/challenge_2.js
const readlineSync = require('readline-sync')

let high = readlineSync.questionInt("Tinggi : ");
let wide = readlineSync.questionInt("Lebar : ");

while (high > 0){
    let temp = "";
	for(let i = 0; i < wide; i++){
		temp += ((i+1)+" ");
    }
    console.log(temp);
	high--;
}

File: dart/challenge_2.dart
void main(){
  
  List<int> temp;
  int tinggi  = 4;
  int lebar   = 6;

  for(int x = 0; x < tinggi; x++){
    temp = [];

    for(int y = 1; y <= lebar; y++){
      temp.add(y);
    }

    print(temp.join(" "));
  }
}